client/state/data-layer/wpcom/sites/rewind/downloads/index.js

```diff
@@ -66,22 +70,112 @@ const fromApi = data => ( {
 	downloadCount: +data.downloadCount,
 	validUntil: data.validUntil,
 	url: data.url,
+	error: data.error,
 } );
 
-export const updateProgress = ( { dispatch }, { siteId, downloadId }, data ) =>
-	dispatch( updateRewindBackupProgress( siteId, downloadId, data ) );
+/**
+ * When requesting the status of a backup creation, it could be done by a specific download Id or without it.
+ * In the second case a collection is returned so we get the first and latest backup creation object.
+ * If an error property was found, it will display an error card stating it.
+ * Otherwise the backup creation progress will be updated.
+ *
+ * @param {function} dispatch Method to trigger a state change.
+ * @param {number}   siteId   Id of the site for the one we're creating a backup.
+ * @param {object}   apiData  Data returned by a successful response.
+ */
+export const updateProgress = ( { dispatch }, { siteId }, apiData ) => {
+	if ( ! isEmpty( apiData[ 0 ] ) ) {
+		const data = fromApi( apiData[ 0 ] );
+		dispatch(
+			get( data, [ 'error' ], false )
+				? rewindBackupUpdateError( siteId, data.downloadId, data )
+				: updateRewindBackupProgress( siteId, data.downloadId, data )
+		);
+	}
+};
 
+/**
+ * If the backup creation progress request fails, an error notice will be shown.
+ *
+ * @param {function} dispatch Method to trigger a state change.
+ *
```

Comment:
small nitpick, but was the blank line here intended?

Reply:
Yes, I always write phpDocs like this since it's their standard, with a blank line before params, another before return, and IMHO it's more legible than not having them

http://usejsdoc.org/tags-returns.html

http://docs.phpdoc.org/getting-started/your-first-set-of-documentation.html#example

Reply:
ah. well in the linked JSDoc article there's no space and I haven't seen any other Calypso code add the space.

how would you feel about taking it out and then if you feel strongly about this, to make a p2 and ask others if they would like to start adding it.

if it's something we all want to do, we can just update everything together so that we don't end up with two standards.